src/lightning_app/runners/cloud.py

```diff
@@ -215,6 +216,18 @@ def dispatch(
                 )
                 works.append(V1Work(name=work.name, spec=work_spec))
 
+                dict_works.append(
```

Comment:
This isn't required once my PR is merged.

Reply:
Oh whoops, that's not required at all, accidental change haha. That's from something else 😆